kolibri/plugins/device/assets/src/views/ManageContentPage/index.vue

```diff
@@ -45,20 +45,23 @@
           @clearall="handleClickClearAll"
         />
 
-        <p v-if="!channelsAreInstalled">
+        <p v-if="!channelsAreInstalled && !channelListLoading">
           {{ $tr('emptyChannelListMessage') }}
         </p>
 
         <div class="channels-list">
-          <ChannelPanel
-            v-for="channel in sortedChannels"
-            :key="channel.id"
-            :channel="channel"
-            :disabled="channelIsBeingDeleted(channel.id)"
-            :showNewLabel="showNewLabel(channel.id)"
-            @select_delete="deleteChannelId = channel.id"
-            @select_manage="handleSelectManage(channel.id)"
-          />
+          <KCircularLoader v-if="channelListLoading" />
```

Comment:
**non-blocking, hoping for additions in a follow-up PR:**

having the loading spinner here & in `Permissions` is **really nice**, and its helpfulness is highlighted in comparison to the behavior in `Info` & `Facilities` within this same plugin, which don't appear to provide any loading indicators. it would be nice to add it in there as well in the future, if possible!

https://github.com/learningequality/kolibri/assets/43046460/5c945457-d582-4c60-925a-6fc7fffba47b